### PR TITLE
Clean Icon Names

### LINKED ISSUE #3701: Cleaning icon names in this repository
As this is a developement repository, it might make more sense to have verbose names for some icons.

I'm thinking specifically about stuff like steam_icon_xxx or chrome-xyz123somethingorother.  
Naming them using a verbose name (For example `steam_icon_281990` would be `stellaris`) and then adding the proper icon name in data.json as a symlink would make editing down the line a lot easier.  
For example, it makes it easier to find if a game already has an icon when GOG or whatever releases a Linux version with another icon name.

We would obviously need to be done with #3481 and #3482 before, to not have to redo these lists as well, but I think it could be useful.

Here is a list of those icons as of April 29 2017:

- [ ] chrome-bikioccmkafdpakkkcpdbppfkghcmihk-Default.svg
- [ ] chrome-gebjgjhbjedcomcajgpodjgfjgkepgpl-Default.svg
- [ ] chrome-dkfhfaphfkopdgpbfkebjfcblcafcmpi-Default.svg
- [ ] chrome-epionbmgciknkbdibokmpmcfllkdfeeo-Default.svg
- [ ] chrome-koegeopamaoljbmhnfjbclbocehhgmkm-Default.svg
- [ ] chrome-mojepfklcankkmikonjlnidiooanmpbb-Default.svg
- [ ] chrome-cnciopoikihiagdjbjpnocolokfelagl-Default.svg
- [ ] chrome-ofbpedjdfkboehhknoohhenbpmgcjoik-Default.svg
- [ ] chrome-omimkinlomnncbmnceacpkmlbfaapojj-Default.svg
- [ ] chrome-pkbfakcphdokdnbdhlndlpcehebmplab-Default.svg
- [ ] chrome-gcjbnclfkeffmdfdhflnadahgmajfgcm-Default.svg
- [ ] chrome-eempgbpnkjnacmilmobpbhbfpdjdcpgd-Default.svg
- [ ] chrome-aeellliejpiclijnghognjagifplmmij-Default.svg
- [ ] chrome-iggpfpnahkgpnindfkdncknoldgnccdg-Default.svg
- [ ] chrome-knpkfcpnjfbniadmfchjpcigfhookhaa-Default.svg
- [ ] chrome-fljalecfjciodhpcledpamjachpmelml-Default.svg
- [ ] chrome-njkkjobcechefaoknodniidfjapgfoco-Default.svg
- [ ] chrome-gbchcmhmhahfdphkhkmpfmihenigjmpp-Default.svg
- [ ] chrome-jfbadlndcminbkfojhlimnkgaackjmdo-Default.svg
- [ ] chrome-menkifleemblimdogmoihpfopnplikde-Default.svg
- [ ] chrome-jklljiahjgoglchglekebfljnmbaleig-Default.svg
- [ ] chrome-cijncchffkmlnfdbnkkfclcbnjcoegjc-Default.svg
- [ ] chrome-pnoffddplpippgcfjdhbmhkofpnaalpg-Default.svg
- [ ] chrome-ocgddccilgpeepgglnlpchkpgamkgmld-Default.svg
- [ ] chrome-fnibmbcdeepaahjmddiihohjanlimlmj-Default.svg
- [ ] chrome-cnocophcbjfiimmnhlhleaooedeheifb-Default.svg
- [ ] chrome-https___stackedit.io_editor.svg
- [ ] chrome-bhmicilclplefnflapjmnngmkkkkpfad-Default.svg
- [ ] chrome-mknafpnfcafklkflhenhaldedggpciao-Default.svg
- [ ] chrome-ijfecbiladpinddbjfodaaiahggomhaf-Default.svg
- [ ] chrome-hjkdhkejcnlmkfdodbkdkelefnkobfif-Default.svg
- [ ] chrome-efjnaogkjbogokcnohkmnjdojkikgobo-Default.svg
- [ ] chrome-https___prezi.com.svg
- [ ] chrome-cmnpffgfpcohhpoddjankjanolcekbni-Default.svg
- [ ] chrome-jeogkiiogjbmhklcnbgkdcjoioegiknm-Default.svg
- [ ] chrome-knkapnclbofjjgicpkfoagdjohlfjhpd-Default.svg
- [ ] chrome-kidnkfckhbdkfgbicccmdggmpgogehop-Default.svg
- [ ] chrome-ohmmkhmmmpcnpikjeljgnaoabkaalbgc-Default.svg
- [ ] chrome-efdcneeepllhjlbejkfnaolelbpdacai-Default.svg
- [ ] chrome-hjbljnpdahefgnopeohlaeohgkiidnoe-Default.svg
- [ ] chrome-fgbjpbdfegnodokpoejnbhnblcojccal-Default.svg
- [ ] chrome-ieicmdpibfnjbmjolkmohnelljmjomoj-Default.svg
- [ ] chrome-fhbjgbiflinjbdggehcddcbncdddomop-Default.svg
- [ ] chrome-jbdlaalbpgnopkpnmoppmccbgkjhfmgh-Default.svg
- [ ] chrome-gighmmpiobklfepjocnamgkkbiglidom-Default.svg
- [ ] chrome-ehjonnoogdmhhidjcldckckmfkpcfefh-Default.svg
- [ ] chrome-mlijpphckdfkmcjclnimmbknefojcaol-Default.svg
- [ ] chrome-clhhggbfdinjmjhajaheehoeibfljjno-Default.svg
- [ ] chrome-eknidjojcplfcgkoffboklcbfhmcdhip-Default.svg
- [ ] chrome-bfimiibpomebbdeomajcknhccgpkjmlh-Default.svg
- [ ] chrome-bnadbgmffcofipfljniafanjcafjlbom-Default.svg
- [ ] chrome-hkgndiocipalkpejnpafdbdlfdjihomd-Default.svg
- [ ] steam_icon_232790.svg
- [ ] steam_icon_252950.svg
- [ ] steam_icon_293780.svg
- [ ] steam_icon_221830.svg
- [ ] steam_icon_60.svg
- [ ] steam_icon_302610.svg
- [ ] steam_icon_275390.svg

### PR DESCRIPTION
**Pull request by Foggalong**
This continues the work of @bilelmoussaoui and finally fixes #3701. Switches out the coded names that Chrome and Steam use for descriptive ones and neatens names in a few other places.

### CODE REVIEW

File: data.json [hunk 1]
```diff
@@ -741,14 +751,18 @@
             "root": "anydo",
             "symlinks": [
                 "any_do",
+                "chrome-ocgddccilgpeepgglnlpchkpgamkgmld-Default"
```
Comment (bilelmoussaoui) [inline data.json]: You have forgot a comma here

Comment (Foggalong) [reply data.json]: I'm so glad you pointed that out, I was looking at the Travis output scratching my head as to what all the errors were 😅

**Comment by Foggalong** [comment]
That was a bit round the houses but we got there! I'll squash merge at the end :joy: But yeah, Steams games are now all descriptive. Fixed the issue with duplicates for `crawl` and `ricochet` by renaming both the originals.

**Comment by Foggalong** [comment]
Chrome apps done! As a point of reference [this site](https://crx.dam.io/) which maintains a near complete [list of all chrome apps ever](https://crx.dam.io/ext/) and [this database of Steam apps](https://steamdb.info/search/?a=app&q=) proved really useful for unobfuscating icon names.

**Comment by Foggalong** [comment]
Think we're all good now @bilelmoussaoui @palob :smile:

**Comment by bilelmoussaoui** [comment]
Looks good to me. There are still few camel case icon names. Shouldn't be renamed too? Especially the AdobeXXX.svg icons

**Comment by Foggalong** [comment]
Oh yeah, will add an extra commit for those!

*Edit: make that several commits. There are way more of these than I thought.*

**Comment by palob** [comment]
Are you sure no symlinks fell by the wayside or should I scrutinise thouroughly?

**Comment by Foggalong** [comment]
I don't think I've missed anything, though to be honest it's going to difficult to check without going through the data.json changes line by line because new symlinks added and duplicate icons were removed. Checking commit by commit might be easier because in theory they don't have any overlap icon wise. If it looks like it'll be too fiddly to do that let me know and I can split this merge up into separate PRs for Steam, Chrome, and CamelCase :smile:

**Comment by palob** [comment]
I'll just do a diff of the resulting theme release comparing to the previous release.

**Review by palob — APPROVED** [review]

**Comment by palob** [comment]
By the way, for an extra dose of consistency: ``liferea`` could be ``rss-blue``.

**Review by bilelmoussaoui — COMMENTED** [review]
Why the  'shine' icon was removed?

**Comment by Foggalong** [comment]
@bilelmoussaoui It was a duplicate of the normal RSS icon but I'm not adding new icons to Shine at the moment. I'd sooner it lose an icon for now and a future maintainer see that Liferea needs a new icon. It's worth noting that through the re-jiggling of names Shine & uTouch did get a load of new icons from these clean names :smile: 

As an aside, discrepancies life this could potentially be fixed with symlinks but again not gonna look too much into that myself whilever uTouch and Shine aren't supported.

**Comment by palob** [comment]
``internet-mail`` and ``Adobeillustrator`` (hardcoded anyway, right?) are missing in the latest release.
Dunno if needed.

**Comment by Foggalong** [comment]
`Adobeillustrator` isn't needed, that's a typo from way back when I originally added these icons. The correct icon name is `AdobeIllustrator` which is not included instead.

`internet-mail` should be included though - I accidentally set `mail_generic` as both the root icon and a symlink rather than `internet-mail` as the root as I'd intended.

Thanks for catching that, will submit a quick fix for it now!

**Pull request merged**